codegen/cmd.go

```diff
@@ -219,17 +222,24 @@ func (c Command) buildPushImage(operator model.Operator) error {
 
 	fullImageName := fmt.Sprintf("%v/%v:%v", image.Registry, image.Repository, image.Tag)
 
-	buildCmd := docker.Command("build", "-t", fullImageName, c.BuildRoot)
-
-	if err := buildCmd.Run(); err != nil {
+	log.Printf("Building docker image %v ...", fullImageName)
+	if err := dockerCommand("build", "-t", fullImageName, c.BuildRoot); err != nil {
 		return err
 	}
 
 	if !build.Push {
 		return nil
 	}
 
-	pushCmd := docker.Command("push", fullImageName)
+	log.Printf("Pushing docker image %v ...", fullImageName)
+
+	return dockerCommand("push", fullImageName)
+}
 
-	return pushCmd.Run()
+func dockerCommand(args ...string) error {
+	cmd := exec.Command("docker", args...)
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = os.Stdout
+	cmd.Env = os.Environ()
+	return cmd.Run()
```

Comment:
I meant create an `Interface` so that we can mock in tests later, not create a wrapper function.

Reply:
this change wasn't to fix your comment, just to be able to see stderr/out

i'll do the comments once CI is working